client/pages/create-profile/index.tsx

```diff
@@ -193,14 +194,7 @@ export default function CreateProfile() {
 
             <Box m="auto">
               <Text size="md">PFP</Text>
-              <Image
-                w="8rem"
-                h="8rem"
-                margin="auto"
-                borderRadius="180px"
-                src="/DevDAO.png"
-                alt="developer"
-              />
+              <AddAvatar src="/DevDAO.png" />
```

Comment:
A page file should not accept any props. Please see my comment above. Basically, if anyone vistis the `create-profile/add-avatar` route, <AddAvatar /> will render automatically since it's the default export from the file named `add-avatar.tsx`.

You should be just redirecting the user to that route.

Reply:
I get that pages shouldn't accept any props, in this case I thought a component was the solution to the issue